cnf-tests/testsuites/validationsuite/cluster/validation.go

```diff
@@ -408,4 +411,84 @@ var _ = Describe("validation", func() {
 		})
 	})
+
+	Context("n3000", func() {
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
+				Skip("Cluster doesn't have intel fpga n3000 nic")
+			}
+		})
+
+		It("should have at least 1 intel n3000 nic", func() {
+			n3000NicExists := false
+		FindN3000Nic:
+			for _, n3000Node := range n3000NodeList.Items {
+				for _, fortville := range n3000Node.Status.Fortville {
+					for _, nic := range fortville.NICs {
+						if strings.Contains(nic.Name, "N3000") {
+							n3000NicExists = true
+							break FindN3000Nic
+						}
+					}
+				}
+			}
+			Expect(n3000NicExists).To(BeTrue(), "No Intel fpga n3000 NICs are presented on the n3000nodes")
+		})
+
+		It("should have a ready deployment n3000-controller-manager", func() {
+			deployment, err := testclient.Client.Deployments(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DeploymentName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			Expect(deployment.Status.ReadyReplicas).To(Equal(int32(1)), "Deployment n3000-controller-manager is not ready")
+		})
+
+		It("should have all the required n3000 operator daemonsets", func() {
+			numberN3000Nodes := int32(len(n3000NodeList.Items))
+			daemonsetDriver, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetDriverName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			daemonsetTelemetry, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetTelemetryName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+			daemonsetN3000Daemon, err := testclient.Client.DaemonSets(utils.N3000DaemonNamespace).Get(context.Background(), utils.N3000DaemonsetN3000DaemonName, metav1.GetOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			Expect(daemonsetDriver.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetDriverName))
+			Expect(daemonsetTelemetry.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetTelemetryName))
+			Expect(daemonsetN3000Daemon.Status.NumberReady).To(Equal(numberN3000Nodes), fmt.Sprintf("Daemonset %s is not ready", utils.N3000DaemonsetN3000DaemonName))
+		})
+	})
+
+	Context("sriov-fec", func() {
+		var sriovFecNodeConfigList = &fecv1.SriovFecNodeConfigList{}
+		var n3000NodeList = &fpgav1.N3000NodeList{}
+
+		BeforeEach(func() {
+			testclient.Client.List(context.Background(), n3000NodeList)
+			if len(n3000NodeList.Items) < 1 {
+				Skip("Cluster doesn't have intel fpga n3000 nic")
+			}
+			testclient.Client.List(context.Background(), sriovFecNodeConfigList)
+			if len(sriovFecNodeConfigList.Items) < 1 {
+				Skip("No sriovsecnodeconfig on the cluster")
+			}
+		})
+
+		It("should have a ready deployment sriov-fec-controller-manager", func() {
```

Comment:
should have a ready deployment for the OpenNESS Operator for Intel FPGA PAC N3000 (Management) operator